### DISCUSSION TITLE
InputText callback

### DISCUSSION THREAD

**Discussion by neutralinsomniac** [AllenDang/giu#96] [Q&A] [answered]
Hi again :)

I'm trying to be notified when the escape key is pressed in an InputText field to cancel editing. I found the ability to use the 'cb' argument to InputTextV, but it looks like the data structure (InputTextCallbackData) that gets passed to the callback has both of its fields un-exported. Is this intentional?

**Comment by AllenDang** [comment, marked as answer]
@neutralinsomniac InputTextCallbackData has public methods to access its internal data.

**Reply by neutralinsomniac** [reply]
Oh my gosh I’m blind. Thank you...
